packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -0,0 +1,154 @@
+          <span class="pf-c-chip__icon">
+            <img src="${experience.assetUrl}" alt="${experience.name} icon" width="20px">
+          </span>
+          <span class="pf-c-chip__text">${experience.name}</span>
+        </div>
+        `)}
+        <hr class="pf-c-divider pf-u-pt-md pf-u-pb-md ${this.experience !== 'Sad' ? 'pf-u-display-none-on-sm' : ''}" />
+        <p class="op-feedback__subtitle pf-u-font-size-md ${this.experience !== 'Sad' ? 'pf-u-display-none-on-sm' : ''}">What is
+          wrong?</p>
+        ${repeat(this.errorList, (error) => html`
+        <div
+          class="pf-c-chip pf-m-draggable op-feedback__chip ${this.experience !== 'Sad' ? 'pf-u-display-none-on-sm' : ''} ${this.error === error.name ? 'op-feedback__chip__active' : ''}"
+          @click="${e => this.error = error.name}" @keydown=${e => (e.key === 'Enter')?  this.error = error.name :''} tabindex="0">
+          <span class="pf-c-chip__text">${error.name}</span>
+        </div>
+        `)}
+        <p class="op-feedback__subtitle pf-u-font-size-md pf-u-pt-md">Summary</p>
+        <textarea id="summary" rows="3" name="summary" @input="${(e: HTMLElementEventMap | any) => this.summary = e.target.value}" placeholder="How can we do better?" class="pf-c-form-control pf-m-resize-vertical" required></textarea>
+        <button class="pf-c-button pf-m-block ${this.experience.length===0 ? 'pf-u-display-none-on-sm' : ''}" type="button" @click="${e => {
+            this.submitFeedback();
+            this.openFeedbackModal = !this.openFeedbackModal;
+            this.openConfirmationModal = !this.openConfirmationModal;
+          }}">Submit</button>
+          <button class="pf-c-button pf-m-block ${this.experience.length!==0 ? 'pf-u-display-none-on-sm' : ''}" type="button" disabled>Submit</button>
+      </form>
+    </dialog>
+    <dialog id="op-feedback__panel" class="op-feedback__panel" .open="${this.openConfirmationModal}">
+      <h2 class="pf-u-text-align-center">Thanks your experience is important to us!</h2>
+      <p class="op-feedback__subtitle pf-u-text-align-center pf-u-font-size-xs pf-u-pb-md">Each time a friend submits a
+        experience, it creates a task for our developer team to resolve it with priority.</p>
+      <hr class="pf-c-divider" />
+      <a href="${this.url}" target="_blank">
+        <p class="pf-u-text-align-center pf-u-p-xs">View Feedback</p>
+      </a>
+      <hr class="pf-c-divider" />
+      <p class="op-feedback__subtitle pf-u-text-align-center pf-u-p-xs" @click="${e => this.openConfirmationModal = false}" @keydown=${e => (e.key === 'Enter')? this.openConfirmationModal = false :'' } tabindex="0">
+        close</p>
+    </dialog>
+    <button id="op-feedback__button" type="button" class="op-feedback__button" @click="${e => {
+            this.openFeedbackModal = !this.openFeedbackModal;
+            this.openConfirmationModal = false;
+          }}">
```

Comment:
can you make a public method to toggle feedback? so the other component can use it. for example, we have a link in footer as `Report an Issue`.

Reply:
Trigger function is ```document.getElementById('opcFeedback').shadowRoot.getElementById('op-feedback__button').click()``` and tag can be ```<opc-feedback id="opcFeedback"></opc-feedback>``` can i take this way

Reply:
Universal function is included now to trigger and updated the documentation with respective to that.